game-core/src/test/java/games/strategy/triplea/delegate/DiceRollTest.java

```diff
@@ -1010,8 +1010,8 @@ void testHeavyBombersLhtr2() {
                 .gameDiceSides(testDelegateBridge.getData().getDiceSides())
                 .territoryEffects(TerritoryEffectHelper.getEffects(germany))
                 .build());
-    assertThat(dice.getRolls(4).get(0).getType(), is(Die.DieType.HIT));
-    assertThat(dice.getRolls(4).get(1).getType(), is(Die.DieType.IGNORED));
+    assertThat(dice.getRolls(4).get(0).getType(), is(Die.DieType.IGNORED));
```

Comment:
The previous code always made the first dice roll be the "HIT" roll and the other be "IGNORED".  The new code now keeps the dice rolls in order so this test had to be updated.